### DISCUSSION TITLE
Support of non root access

### DISCUSSION THREAD

**Discussion by JD-ISWT** [Secure-Compliance-Solutions-LLC/GVM-Docker#343] [Q&A]
I deployed the image behind reverse proxy and I search for a way to setup the access throught /openvas

The problem for now is that all returned paths are relatives to /

```
<script src="/static/js/runtime-main.7103e5d7.js"></script> 
<script src="/static/js/2.ea51a949.chunk.js"></script>
```

**Comment by Dexus** [comment]
you need to rewrite the path via reverseproxy

**Reply by JD-ISWT** [reply]
Yes, but for the links sent bye GVM it doesn't works

From my example: 
> /static/js/runtime-main.7103e5d7.js

Here `/static` will not be catched by my reverse proxy config (which works on location `/openvas` )

**Reply by Dexus** [reply]
https://stackoverflow.com/questions/30350185/nginx-reverse-proxy-configuration-to-handle-absolute-paths
